src/ui/atoms/languageSwitcher/languageSwitcher.scss

```diff
@@ -8,25 +8,24 @@
       'language'
       'language';
   }
-}
 
-.languages-translation {
-  color: white;
-  grid-area: languages;
-  white-space: nowrap;
-}
+  .languages-translation {
+    grid-area: languages;
+    white-space: nowrap;
```

Comment:
Is this necessary? I don't see its use

Reply:
Exactly, now it has become unnecessary with typography usage.